magda-web-client/src/config.ts

```diff
@@ -11,10 +11,31 @@ declare global {
     }
 }
 
+// https://www.w3.org/TR/NOTE-datetime
+const defaultDateFormats: string[] = [
+    // Year
+    "YYYY",
+    // Year and month (eg 1997-07)
+    "YYYY-MM",
+    // Complete date (eg 1997-07-16):
+    "DD-MM-YYYY",
+    "MM-DD-YYYY",
+    "YYYY-MM-DD",
+    // Complete date plus hours and minutes (eg 1997-07-16T19:20+01:00):
+    "YYYY-MM-DDThh:mmTZD",
+    // Complete date plus hours, minutes and seconds (eg 1997-07-16T19:20:30+01:00):
+    "YYYY-MM-DDThh:mm:ssTZD",
+    // Complete date plus hours, minutes, seconds and a decimal fraction of a second (eg 1997-07-16T19:20:30.45+01:00)
+    "YYYY-MM-DDThh:mm:ss.sTZD",
+    // Natural language date formats (eg 12 March, 1997)
+    "DD-MMM-YYYY",
+    "MMM-DD-YYYY"
+];
+
 // Local minikube/docker k8s cluster
-// const fallbackApiHost = "http://localhost:30100/";
+const fallbackApiHost = "http://192.168.99.110:30100/";
```

Comment:
Eek, did you mean to leave this in - it'll mean that `yarn run dev` in `magda-web-client` requires minikube right?

Reply:
Nope, good catch!